skaffold.yaml

```diff
@@ -80,18 +80,13 @@ profiles:
   - op: replace
     path: /build/artifacts/7/docker/dockerfile
     value: Dockerfile.debug
-# "networkpolicy" profile adds network policies. By design, the default minikube cni flannel does not
-# support networkpolicies. Use an alternative CNI like calico
-# 'minikube start --cni=calico'
```

Comment:
**My rationale for this change:**
I want to avoid doubling the maintenance cost of documentation.
The documentation you've added to `kustomize/components/network-policies/README.md` is perfecto! :D